examples/admin_create_acls/admin_create_acls.go

```diff
@@ -0,0 +1,142 @@
+// Create ACLs
+package main
+
+/**
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+)
+
+// Parses a list of 7n arguments to a slice of n AclBinding
+func parseAclBindings(args []string) (aclBindings kafka.AclBindings, err error) {
+	nAclBindings := len(args) / 7
+	parsedAclBindings := make(kafka.AclBindings, nAclBindings)
+
+	for i := 0; i < nAclBindings; i += 1 {
+		start := i * 7
+		resourceTypeString := args[start]
+		name := args[start+1]
+		resourcePatternTypeString := args[start+2]
+		principal := args[start+3]
+		host := args[start+4]
+		operationString := args[start+5]
+		permissionTypeString := args[start+6]
+
+		var resourceType kafka.ResourceType
```

Comment:
Don't think you need to predeclare these, just use type inference in the assignment.

`resourceType, err := kafka.ResourceTypeFromString(...)`

Reply:
I think it'll allow a sub-set of left-side variables to already be declared, such as err.

Reply:
ok, thanks. It was giving error but for another reason